app/pages/sections/catalogue-solutions/order-item/contextCreator.js

```diff
@@ -1,9 +1,10 @@
 import manifest from './manifest.json';
 import { baseUrl } from '../../../../config';
+import { getSectionErrorContext } from '../../getSectionErrorContext';
 
 export const populateEstimationPeriod = ((selectedPrice) => {
-  manifest.questions.estimationPeriod.options.forEach((option, i) => {
-    manifest.questions.estimationPeriod.options[i]
+  manifest.questions[2].options.forEach((option, i) => {
```

Comment:
😢 It's a shame that having our manifest with the question as a dictionary rather than an array is so much work. I don't like hardcoding these number. If in a few months a decision is made to add another question before this then this part will fall apart. Can you create a helper function within the helpers folder to find the question in the manifest passing in the questionId. Hopefully that way we can avoid using these hardcoded numbers.